snippets/markdown.json

```
@@ -249,5 +249,32 @@
 				"   {{< nextlink href=\"link/here\" >}}Link desctiption goes here{{< /nextlink >}}",
 				"{{< /whatsnext >}}"
 			]
+	},
+	"Eng Blog Frontmatter":{
+		"prefix": ";;eng_blog_frontmatter",
+		"body": [
+			"---",
+			"authors:",
+			"- email: author@datadoghq.com",
+			"  name: author-full-name",
+			"  image: author-headshot-image.jpg",
+			"blog: engineering",
+			"date: YYYY-MM-DD",
+			"publishDate: YYYY-MM-DD",
+			"description: \"A-short-summary-of-the-post-to-be-used-on-the-front-page-of-the-blog\"",
+			"specific_interest:",
+			"core_product:",
+			" - product-name",
+			"disable_hiring_blurb: false",
+			"draft: false",
+			"private: false",
+			"slug: identical-to-your-markdown-filename",
+			"series: if-the-post-is-one-in-a-series-add-series-identifier",
+			"title: \"title-of-the-blog\"",
```

```suggestion
			"title: \"Title of the blog\"",
```
Or in title case, if that's the style.

---

Thanks!